lib/vtt.js

```diff
@@ -1311,16 +1311,51 @@
         }
       }
 
-      // 3.2 WebVTT metadata header syntax
-      function parseHeader(input) {
-        parseOptions(input, function (k, v) {
-          switch (k) {
-          case "Region":
-            // 3.3 WebVTT region metadata header syntax
-            parseRegion(v);
+      // draft-pantos-http-live-streaming-20
+      // https://tools.ietf.org/html/draft-pantos-http-live-streaming-20#section-3.5
+      // 3.5 WebVTT
+      function parseTimestampMap(input) {
+        var settings = new Settings();
+
+        parseOptions(input, function(k, v) {
+          switch(k) {
+          case "MPEGT":
+            settings.integer(k + 'S', v);
+            break;
+          case "LOCA":
+            settings.set(k + 'L', parseTimeStamp(v));
             break;
           }
-        }, /:/);
+        }, /[^\d]:/, /,/);
+
+        self.ontimestampmap && self.ontimestampmap({
```

Comment:
Yes, the user would have to manually shift the timestamps. The timestamp map tag only gives a mapping from cue time to TS time, but the mapping from TS to display time is still needed for the shift. I think this approach is the easiest instead of storing state on the parse for the extra mapping